src/services/expenses.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+let expenses = [];
+
+const getExpenses = (filterProperties) => {
+  const { userId, categories, from, to } = filterProperties;
+
+  switch (true) {
+    case userId:
+      return expenses.find(expanse => expanse.userId === userId);
+    case categories:
+      return expenses.find(expanse => expanse.categories.includes(categories));
+    case from:
+      return expenses.find(expanse => expanse.spentAt >= from);
+    case to:
+      return expenses.find(expanse => expanse.spentAt <= to);
```

Comment:
you must filter the entire array of expenses and return all of them that match the criteria